### DISCUSSION TITLE
Boot or startup time rather slow

### DISCUSSION THREAD

**Discussion by sIRwa2** [mifi/lossless-cut#817] [Support]
opening a 4,46Gb mp4 = 13 seconds (312 frames) in Lossless-cut
VLC = 19 frames 

is there a way to win back some boot speed?

**Comment by mifi** [comment]
Hi. Are you talking about the time it takes the losslesscut app to start up when launching it, or the time it takes to load a file? What kind of file are you trying to load? Normally loading a file should take just a second. could you post the report (help menu), that might help me understand why it's taking so long

**Comment by sIRwa2** [comment]
i always open the program with the "open with" in the rightmouse menu. The files are recorded with the NVidia shadowplay feature, Gameplay recordings. Log: https://controlc.com/f5f22f38

**Reply by mifi** [reply]
maybe you can try to drag-drop a file into an existing open losslesscut window instead, see if that's faster?

**Reply by sIRwa2** [reply]
HI, sorry for the slow replies, I installed 3.38.0 I also used the zip and unpacked it to my program files instead of the exe.
opening the program is pretty fast: 

2 Sec 19 Frames (recorded at 24 f/s) 

drag onto the program is:
5 sec and 21 frames

together its:
8 sec  16 frames

old fashion right mouse: open with:
8 sec 08 frames

I guess the zip version and or the newer version is a tad quicker?

**Reply by mifi** [reply]
I think the exe download is self-extracting, so that's why it's slower, as it needs to extract when starting up
